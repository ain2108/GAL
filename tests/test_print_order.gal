int main(){
	print_int(6);
	print_endline();
	print_str("Hello");
	print_endline();
	print_int(903);
	print_endline();
    return 1;
}