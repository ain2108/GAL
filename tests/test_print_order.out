6
Hello
903